.env.charlie

```
@@ -0,0 +1,39 @@
+# Charlie Build Environment Variables
+# This file contains dummy values for Charlie to run production builds
+# in ephemeral devbox environments without real credentials
+
+# Next.js Public Variables (dummy values for build)
+NEXT_PUBLIC_SUPABASE_URL="https://dummy.supabase.co"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)